src/backend/base/langflow/components/data/file.py

```diff
@@ -31,6 +123,57 @@ class FileComponent(BaseFileComponent):
 
     inputs = [
         *_base_inputs,
+        BoolInput(
+            name="advanced_mode",
+            display_name="Advanced Parser",
+            value=False,
+            real_time_refresh=True,
+            info=(
+                "Enable advanced document processing and export with Docling for PDFs, images, and office documents. "
+                "Available only for single file processing. Requires installation: uv pip install docling"
+            ),
+            show=False,
+        ),
+        DropdownInput(
+            name="pipeline",
+            display_name="Pipeline",
+            info="Docling pipeline to use",
+            options=["standard", "vlm"],
+            value="standard",
+            advanced=True,
+        ),
+        DropdownInput(
+            name="ocr_engine",
+            display_name="OCR Engine",
+            info="OCR engine to use. Only available when pipeline is set to 'standard'.",
+            options=["", "easyocr"],
+            value="",
+            advanced=True,
```

Comment:
I think you missed this. I like the `show=False` approach, by the way!
```suggestion
            value="",
            show=False,
            advanced=True,
```